source/CIGameScene.cpp

```diff
@@ -80,7 +91,7 @@ bool GameScene::init(const std::shared_ptr<cugl::AssetManager>& assets,
     _gameUpdateManager = GameUpdateManager::alloc();
     _networkMessageManager = networkMessageManager;
     _networkMessageManager->setGameuUpdateManager(_gameUpdateManager);
-    if (isHost) {
+    if (gameId.empty()) { // Host
```

Comment:
This is not a good way to determine whether this player is the host. Player id being 0 means the player is the host, theres a method for this in network message manager